docs/contributing/upgrading-dependencies.md

```diff
@@ -3,6 +3,22 @@
 This document outlines how we manage the dependencies that GitHub Desktop needs,
 as some need to be handled differently to others
 
+## General Guidelines
+
+In the interest of stability and caution we tend to stay a version (or more) behind on our most critical dependencies. We vary whether this is a major, minor, or patch version depending on the development practices of the dependency's project.
+
+| Dependency  | Versions Behind Latest |
+| --- | --- |
+| electron | >= 1 major |
+| electron-builder | >= 1 major |
+| electron-packager | >= 1 major |
```

Comment:
👍 this one revs enough the major version is incremented to indicate breaking changes